tox.ini

```diff
@@ -21,14 +21,17 @@ covercmd =
       --cover-branches \
       --nocapture
 
-[testenv:py27]
-basepython =
-    python2.7
-deps =
-    {[testenv]deps}
-    grpcio >= 0.13.0
-setenv =
-    PYTHONPATH =
+# Until grpcio 0.13.1 ships, this environment is broken on UCS2 builds.
+# See: https://github.com/grpc/grpc/issues/5280 and
+#      https://github.com/grpc/grpc/pull/5319
```

Comment:
I would normally agree, except that they [intend to release a fix early next week](https://github.com/grpc/grpc/pull/5319#issuecomment-192412023).